modules/gcloud/go.mod

```diff
@@ -10,6 +10,7 @@ require (
 	cloud.google.com/go/pubsub v1.36.2
 	cloud.google.com/go/spanner v1.57.0
 	github.com/docker/go-connections v0.5.0
+	github.com/stretchr/testify v1.9.0
```

Comment:
I think we can live without testify. It's indeed used in other modules, but we can keep this one without it for now. Wdyt?

Reply:
Yeah this shouldn't be needed